main/src/org/jetbrains/bsp/bazel/common/Constants.java

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.bsp.bazel.common;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+
+public class Constants {
```

Comment:
do we want to keep it in the class or move it to the `.properties` file? https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html

Reply:
Maybe I'll leave it as it is for now, reading properties will require a lot of parsing and/or typecasting and it won't provide any value for us, because these constants are non-configurable anyway

Reply:
okey, but I'll change it in the future